src/hayhooks/server/utils/mcp_utils.py

```diff
@@ -18,12 +18,18 @@
 from haystack.lazy_imports import LazyImport
 from hayhooks.server.routers.deploy import PipelineFilesRequest
 from fastapi.concurrency import run_in_threadpool
+from contextlib import asynccontextmanager
+from starlette.applications import Starlette
+from starlette.routing import Mount, Route
+from starlette.types import Receive, Scope, Send
+from typing import AsyncIterator
 
 
 with LazyImport("Run 'pip install \"mcp\"' to install MCP.") as mcp_import:
     from mcp.types import Tool, TextContent, ImageContent, EmbeddedResource
     from mcp.server import Server
-
+    from mcp.server.streamable_http_manager import StreamableHTTPSessionManager
```

Comment:
IIRC - lock mcp-sdk dependency on mcp-sdk > 1.8.0 because of this import - if not already locked.

Reply:
Yeah good point